src/commands/deploy/nextjs/deploy.ts

```diff
@@ -96,6 +97,13 @@ export async function nextJsDeploy(options: GenezioDeployOptions) {
     log.info(
         `The app is being deployed at ${colors.cyan(cdnUrl)}. It might take a few moments to be available worldwide.`,
     );
+
+    await handleEnvVars(
+        options.env,
+        deploymentResult.projectId,
+        deploymentResult.projectEnvId,
+        process.cwd(),
+    );
```

Comment:
Is there a reason why not do this in the `Promise.all` above the log? Or maybe inside `setupEnvironmentVariables`?